apps/console/src/features/identity-providers/components/settings/authenticator-settings.tsx

```diff
@@ -124,15 +136,74 @@ export const AuthenticatorSettings: FunctionComponent<IdentityProviderSettingsPr
         useState<FederatedAuthenticatorWithMetaInterface[]>([]);
     const [ availableTemplates, setAvailableTemplates ] =
         useState<IdentityProviderTemplateInterface[]>(undefined);
-    const [ availableManualModeOptions, setAvailableManualModeOptions ] =
-        useState<FederatedAuthenticatorMetaDataInterface[]>(undefined);
+    const [
+        availableManualModeOptions,
+        setAvailableManualModeOptions
+    ] = useState<FederatedAuthenticatorMetaDataInterface[]>(undefined);
     const [ showAddAuthenticatorWizard, setShowAddAuthenticatorWizard ] = useState<boolean>(false);
-    const [ isTemplatesLoading, setIsTemplatesLoading ] = useState<boolean>(false);
-    const [ isPageLoading, setIsPageLoading ] = useState<boolean>(true);
     const [ isSubmitting, setIsSubmitting ] = useState<boolean>(false);
+    const [ accordionActiveIndexes, setAccordionActiveIndexes ] = useState<number[]>([]);
+    const [ isIdPTemplateFetchRequestLoading, setIdPTemplateFetchRequestLoading ] = useState<boolean>(undefined);
+    const [
+        isFederatedAuthenticatorFetchRequestLoading,
+        setFederatedAuthenticatorFetchRequestLoading
+    ] = useState<boolean>(undefined);
 
     const config: ConfigReducerStateInterface = useSelector((state: AppState) => state.config);
 
+    const isActiveTemplateExpertMode: boolean = useMemo(() => {
+        return identityProviderConfig?.templates?.expertMode &&
+            (identityProvider.templateId === ExpertModeIdpTemplate.id || !identityProvider.templateId);
+    }, [ identityProvider, identityProviderConfig  ]);
+
+    /**
+     * When `availableAuthenticators` updates, filter the templates.
+     */
+    useEffect(() => {
+
```

Comment:
```suggestion
```

Reply:
Actually added the padding intentianally. IMO, we can adopt WSO2 method padding for Java here as well for better readability. WDYT?

Unfortunately I couldn't find a matching ESLint rule for this use case, since [`padded-blocks`](https://eslint.org/docs/latest/rules/padded-blocks) adds a padding to the end of the function as well.